apps/global/i18n/translations/en.json

```diff
@@ -106,6 +106,11 @@
     "meetTheCollectiveTitle": "Meet The Collective",
     "meetTheCollectiveDescription": "Dream On: Speaker Collective is a team of authors, speakers, and preachers. We are passionate about empowering and teaching others to pursue their dreams and purposes God has put in each and everyone’s life."
   },
+  "board": {
+    "pageTitle": "Dream On: Board Collective",
+    "meetTheCollectiveTitle": "Meet The Collective",
+    "meetTheCollectiveDescription": "Dream On: Board Collective is a team of authors, speakers, and preachers. We are passionate about empowering and teaching others to pursue their dreams and purposes God has put in each and everyone’s life."
```

Comment:
Let's get rid of this all together. Not sure what I want to put for this at the moment and we can always add it back in the future